src/main/java/org/kiwiproject/test/junit/jupiter/MongoServerExtension.java

```diff
@@ -102,11 +106,17 @@
     @Getter
     private final DropTime dropTime;
 
+    /**
+     * The version of the mongo server to use;
```

Comment:
I am guessing you forgot to add a note about the default version after the semicolon?

Reply:
I didn't mean to put the semicolon but I didn't think I needed to put anything in this doc for the default.

Reply:
OK, but then how does someone who is only using the Javadoc know what Mongo version is used if not explicitly specified? They would need to read the code, navigate to the `MongoServerTests` utility class, and then read about it.  Obviously it's not a huge deal, but wouldn't it be nicer to just mention it somewhere in this extension?

Reply:
And then the question is whether it should be added on the field javadoc, or at the class-level, or maybe actually in the constructors that do _not_ accept a `ServerVersion`

Reply:
Sorry, maybe we should put that information at both the class-level _and_ those two constructors that do _not_ accept `ServerVersion`.

Reply:
I see your point, however, we have a default for `DropTime` and don't have a javadoc for that specifying.

Reply:
Ok, I stand corrected, it is in the class level javadoc

Reply:
I'll make a couple of updates

Reply:
Pushed changes